nvflare/private/fed/client/client_engine_executor_spec.py

```diff
@@ -20,6 +20,53 @@
 class ClientEngineExecutorSpec(ClientEngineSpec):
     """The ClientEngineExecutorSpec defines the ClientEngine APIs running in the child process."""
 
+    def register_aux_message_handler(self, topic: str, message_handle_func):
+        """Register aux message handling function with specified topics.
+
+        Exception is raised when:
+            a handler is already registered for the topic;
+            bad topic - must be a non-empty string
+            bad message_handle_func - must be callable
+
+        Implementation Note:
+        This method should simply call the ClientAuxRunner's register_aux_message_handler method.
+
+        Args:
+            topic: the topic to be handled by the func
+            message_handle_func: the func to handle the message. Must follow aux_message_handle_func_signature.
+
+        """
+        pass
+
+    def send_aux_request(self, topic: str, request: Shareable, timeout: float, fl_ctx: FLContext) -> Shareable:
+        """Send a request to Server via the aux channel.
+
+        Implementation: simply calls the ClientAuxRunner's send_aux_request method.
+
+        Args:
+            topic: topic of the request
+            request: request to be sent
+            timeout: number of secs to wait for replies. 0 means fire-and-forget.
+            fl_ctx: FL context
+
+        Returns: a reply Shareable
+
+        """
+        pass
+
+    def fire_and_forget_aux_request(self, topic: str, request: Shareable, fl_ctx: FLContext) -> Shareable:
```

Comment:
This was a helper function.
We need to check its usage and definition to see if we require this method to be implemented by the child classes